src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -492,8 +489,8 @@ void DlgPrefEQ::slotQuickEffectChangedOnDeck(int effectIndex) {
     EffectChainPointer pChain = m_pEffectsManager->getEffectChain(unitGroup);
     QList<EffectChainPresetPointer> presetList =
             m_pChainPresetManager->getQuickEffectPresetsSorted();
-    if (pChain && effectIndex > 0 && effectIndex <= presetList.size()) {
-        pChain->loadChainPreset(presetList[effectIndex - 1]);
+    if (pChain && effectIndex > -1 && effectIndex <= presetList.size()) {
```

Comment:
```suggestion
    if (pChain && effectIndex >= 0 && effectIndex < presetList.size()) {
```
looks more pleasant IMHO